fabfile.py

```diff
@@ -30,7 +30,7 @@
 SETTINGS_DIR = "goldstone.settings"
 
 # The default settings are to run Elasticsearch and PostgreSQL locally.
-DEV_SETTINGS = SETTINGS_DIR + ".local_docker"
+DEV_SETTINGS = SETTINGS_DIR + ".local_dev"
```

Comment:
I deleted my previous goldstone-server tree, so I'll have to recreate this file.   It should be checked into the repository.